### ISSUE TITLE
Linux Wiki Page References develop3d

### ISSUE THREAD

**Issue by ashes999** [mono/MonoGame#3563]
Hi,

While reading [this Linux getting-started wiki page](https://github.com/mono/MonoGame/wiki/Tutorials:-Getting-Started-on-Linux). The tutorial itself never references any branch/tag, and I don't see any branch/tag in the repository named `develop3d`. I see several closed issues, and some open ones from 2012-2013.

Is this a branch that was merged into `master` and no longer exists?

**Comment by tomspilman** [comment]
That wiki page is pretty old... in fact most of the wiki is old.

@craz0y - Is that page something that is useful to translate to the new docs?

**Comment by ashes999** [comment]
@tomspilman can you point me to a more current "getting started on Linux" doc? This is the top link from Google.

**Comment by tomspilman** [comment]
If you go here:

http://www.monogame.net/documentation/?page=Introduction

.. and look under "Setting Up MonoGame" you will see some info on Linux.

**Comment by ashes999** [comment]
Noted, thanks.

**Comment by cra0zy** [comment]
2 things to note:
1) Since the build server doesn't automatically build monogame-linux.run, get it from here: https://mega.co.nz/#F!yEIHGLjS!OvjruaOdTQmgmTY4RvXLZw
2) The monogame-linux.run package only includes Monogame Pipeline, get the monodevelop extension from: https://github.com/mono/MonoGame/pull/3361 Once the extension is complete I will include it in the installer

**Comment by ashes999** [comment]
@cra0zy I'm completely new to Mono and MonoGame. Can you explain what exactly this is?

It seems from #3361 that we're talking about new project templates. Any approximate ETA on that?

**Comment by cra0zy** [comment]
Scroll to the bottom, there is a download link.

**Comment by cra0zy** [comment]
Also, I explained how to install monogame 3.3 this morning here: community.monogame.net/t/windows-to-linux-port-game-runs-too-fast/2005/5

**Comment by ashes999** [comment]
I'll check it out.

Currently, I'm getting [this same error](http://stackoverflow.com/questions/25171606/monogame-application-says-sdl-dll-is-missing-even-though-its-there-why) about `SDL.dll` missing. I'll see if installing/reinstalling with your instructions fixes it. @cra0zy
